digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -1046,6 +1048,19 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
     speedLimitService.getUnknown(includedMunicipalities)
   }
 
+  get("/speedLimits/qualityErrors") {
+    val user = userProvider.getCurrentUser()
+    val municipalityCode = user.configuration.authorizedMunicipalities
+    municipalityCode.foreach(validateUserMunicipalityAccess(user))
+
+    user.isOperator() match {
+      case true =>
+        speedLimitService.getSpeedLimitsWithQualityErrors()

```

Comment:
why is it called `quality errors` ?

Reply:
We can start having the same name suggested on other comment